packages/plugin/webpack/test/WebpackConfig_spec.ts

```diff
@@ -483,7 +331,7 @@ describe('WebpackConfigGenerator', () => {
       } as WebpackPluginConfig;
       const generator = new WebpackConfigGenerator(config, mockProjectDir, false, 3000);
       const webpackConfig = await generator.getRendererConfig(config.renderer.entryPoints);
-      expect(webpackConfig[0].target).to.equal('electron-preload');
+      expect(webpackConfig[0].target).to.equal('web');
```

Comment:
this test uses an entry with `nodeintegration: false` (default) , so witih the new changes it gets put into `sandboxedpreload` bucket which resolves to `web` target rather than `electron-preload` (unsandboxed preloads)